package.json

```diff
@@ -114,6 +118,7 @@
     "phantomjs-prebuilt": "2.1.4",
     "react-addons-test-utils": "0.14.3",
     "react-hot-loader": "1.3.0",
+    "react-imageloader": "2.1.0",
```

Comment:
What determines if a `react-*` module is a dependency or a devDependency? Are all of those `react-*` modules above runtime deps?